compiler/rustc_abi/src/lib.rs

```diff
@@ -1241,6 +1241,31 @@ impl Abi {
     pub fn is_scalar(&self) -> bool {
         matches!(*self, Abi::Scalar(_))
     }
+
+    /// Returns the fixed alignment of this ABI, if any
+    pub fn inherent_align<C: HasDataLayout>(&self, cx: &C) -> Option<AbiAndPrefAlign> {
```

Comment:
```suggestion
    /// Returns the fixed alignment of this ABI, if any is mandated.
    pub fn inherent_align<C: HasDataLayout>(&self, cx: &C) -> Option<AbiAndPrefAlign> {
```
(cc @RalfJung, it would be good to use this same method in the sanity check, right?)

Reply:
Added & updated layout_sanity_check.